### PR TITLE
enable MasterData/IntegrationEvents IntegrationTests in CI

### PR DESCRIPTION
**Pull request by lasrinnil**
## Description
These IntegrationTests project were never implanted in the CI which was a major blunder. The PR will introduce the `MasterData` and `IntegrationEvents` integrationtests into the CI

## References
https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-mosaic/308
## Checklist
- [ ] Should the change be behind a feature flag?
- [ ] Can the feature be meaningfully disabled or circumvented if there are issues (e.g., database-breaking changes)?
- [ ] Has it been considered whether data is being delivered to the wrong actor?
- [ ] Subsystem test executed (dev_002/dev_003)
- [ ] Is there time to monitor state of the release to Production?
- [ ] Reference to the task

### CODE REVIEW

File: .github/workflows/ci-dotnet.yml [hunk 1]
```diff
@@ -94,6 +94,18 @@ jobs:
             use_azure_functions_tools: true
             contentroot_variable_name: empty # Means skip
 
+          - name: IntegrationEvents - integration tests
+            paths: \source\IntegrationEvents.IntegrationTests\bin\Release\net8.0\Energinet.DataHub.EDI.IntegrationEvents.IntegrationTests.dll
+            filter: empty # Means skip
```
Comment (MadsDue) [inline .github/workflows/ci-dotnet.yml:99]: Does this mean that the tests will be skipped?

File: .github/workflows/ci-dotnet.yml [hunk 2]
```diff
@@ -94,6 +94,18 @@ jobs:
...
+            use_azure_functions_tools: true
+            contentroot_variable_name: empty # Means skip
```
Comment (MadsDue) [inline .github/workflows/ci-dotnet.yml:101]: same

File: .github/workflows/ci-dotnet.yml [hunk 3]
```diff
@@ -94,6 +94,18 @@ jobs:
...
+
+          - name: MasterData - integration tests
+            paths: \source\MasterData.IntegrationTests\bin\Release\net8.0\Energinet.DataHub.EDI.MasterData.IntegrationTests.dll
+            filter: empty # Means skip
```
Comment (MadsDue) [inline .github/workflows/ci-dotnet.yml:105]: same

File: .github/workflows/ci-dotnet.yml [hunk 4]
```diff
@@ -94,6 +94,18 @@ jobs:
...
+            use_azure_functions_tools: true
+            contentroot_variable_name: empty # Means skip
```
Comment (MadsDue) [inline .github/workflows/ci-dotnet.yml:107]: same

**Review by MadsDue — APPROVED** [review]

**Review by kerbou — APPROVED** [review]

**Pull request merged**